src/amber/cli/commands/encrypt.cr

```diff
@@ -0,0 +1,41 @@
+require "../../support/message_encryptor"
+
+module Amber::CLI
+  class MainCommand < ::Cli::Supercommand
+    command "e", aliased: "encrypt"
+
+    class Encrypt < ::Cli::Command
+      class Options
+        arg "env", desc: "environment file to encrypt", default: "production"
+        string ["-e", "--editor"], desc: "editor", default: "vim"
+      end
+
+      class Help
+        caption "# Encrypts Environment"
```

Comment:
I think this is a really nice feature, why I think we should add nice docs for the command.

Reply:
Of course. We need nice docs for a lot of things.

Reply:
What I was stating is that we should write/explain how to use this command when running `amber e --help`